spring-grpc-docs/src/main/antora/modules/ROOT/partials/_configprops.adoc

```diff
@@ -2,22 +2,22 @@
 |Name | Default | Description
 
 |spring.grpc.client.channels |  | 
-|spring.grpc.client.default-channel.address | `+++static://localhost:9090+++` | The target address uri to connect to.
-|spring.grpc.client.default-channel.default-load-balancing-policy | `+++round_robin+++` | The default load balancing policy the channel should use.
-|spring.grpc.client.default-channel.enable-keep-alive | `+++false+++` | Whether keep alive is enabled on the channel.
-|spring.grpc.client.default-channel.health.enabled | `+++false+++` | Whether to enable client-side health check for the channel.
-|spring.grpc.client.default-channel.health.service-name |  | Name of the service to check health on.
-|spring.grpc.client.default-channel.idle-timeout | `+++20s+++` | The duration without ongoing RPCs before going to idle mode.
-|spring.grpc.client.default-channel.keep-alive-time | `+++5m+++` | The delay before sending a keepAlive. Note that shorter intervals increase the network burden for the server and this value can not be lower than 'permitKeepAliveTime' on the server.
-|spring.grpc.client.default-channel.keep-alive-timeout | `+++20s+++` | The default timeout for a keepAlives ping request.
-|spring.grpc.client.default-channel.keep-alive-without-calls | `+++false+++` | Whether a keepAlive will be performed when there are no outstanding RPC on a connection.
-|spring.grpc.client.default-channel.max-inbound-message-size | `+++4194304B+++` | Maximum message size allowed to be received by the channel (default 4MiB). Set to '-1' to use the highest possible limit (not recommended).
-|spring.grpc.client.default-channel.max-inbound-metadata-size | `+++8192B+++` | Maximum metadata size allowed to be received by the channel (default 8KiB). Set to '-1' to use the highest possible limit (not recommended).
-|spring.grpc.client.default-channel.negotiation-type | `+++plaintext+++` | The negotiation type for the channel.
-|spring.grpc.client.default-channel.secure | `+++true+++` | Flag to say that strict SSL checks are not enabled (so the remote certificate could be anonymous).
-|spring.grpc.client.default-channel.ssl.bundle |  | SSL bundle name.
-|spring.grpc.client.default-channel.ssl.enabled |  | Whether to enable SSL support. Enabled automatically if "bundle" is provided unless specified otherwise.
-|spring.grpc.client.default-channel.user-agent |  | The custom User-Agent for the channel.
+|spring.grpc.client.default-channel.address |  | 
```

Comment:
I need to look into why this is not finding the default any longer. I have an idea but you can ignore this part of the code for the review for now.

Reply:
@dsyer I manually adjusted the props for now so as to not block this PR merge. I will fix this in subsequent PR.

Reply:
Tracked here https://github.com/spring-projects-experimental/spring-grpc/issues/92